docs/user/vsphere/post.md

```diff
@@ -0,0 +1,9 @@
+# (Optional) Unassign Privileges
+Once an install has completed, the cluster does not require any user privileges. If a user was granted adminstrative or escalated privileges, those privileges can be safely removed, if desired. Keep in mind, that the privileges would be required in order to destroy the cluster.
```

Comment:
The user that you are providing for installation will also be used for cloud-config.  That user needs to create disks for PVs. You won't be able to remove it.

https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/vcp-roles.html